source/agora/test/Base.d

```diff
@@ -95,7 +95,10 @@ public import std.range;
 // To print messages to the screen while debugging a test
 public import std.stdio;
 
+// Convenience constants
 public const size_t GenesisValidators = GenesisBlock.header.enrollments.count();
+public const uint GenesisValidatorCycle = GenesisBlock
+    .header.enrollments[0].cycle_length;
```

Comment:
Not a big deal, but at global scope, use `immutable` instead of `const`.
They are functionally equivalent, it's simply a matter of style.